usaspending_api/etl/subaward_etl.py

```diff
@@ -180,9 +180,15 @@ def load_subawards(submission_attributes, db_cursor):
         recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
 
         if not recipient and row['duns']:
+            recipient_name = row['awardee_name']
+            if recipient_name is None:
+                recipient_name = row['awardee_or_recipient_legal']
+            if recipient_name is None:
```

Comment:
No? If it is "" we allow it to be stored as the blank string as long as the duns exist, or am I misunderstanding here?